example/90-shoot-packet.yaml

```diff
@@ -0,0 +1,132 @@
+---
+apiVersion: garden.sapcloud.io/v1beta1
+kind: Shoot
+metadata:
+  name: johndoe-packet
+  namespace: garden-dev
+spec:
+  cloud:
+    profile: packet
+    region: EWR1
+    secretBindingRef:
+      name: core-packet
+    packet:
+      networks:
```

Comment:
can we leave this section out if nothing can be configured here (other than pod/service/cidr)?

Reply:
Sure. Updating now in the template and regenerating.